src/ui/flutter_app/lib/common/config.dart

```diff
@@ -100,119 +100,131 @@ class Config {
   static int endgameNMoveRule = 100;
   static bool threefoldRepetitionRule = true;
 
+  // TODO: use jsonSerializable
   static Future<void> loadSettings() async {
     print("[config] Loading settings...");
 
     final settings = await Settings.instance();
 
     Config.isPrivacyPolicyAccepted =
-        settings['IsPrivacyPolicyAccepted'] ?? false;
+        settings['IsPrivacyPolicyAccepted'] as bool? ?? false;
 
     // Preferences
-    Config.toneEnabled = settings['ToneEnabled'] ?? true;
-    Config.keepMuteWhenTakingBack = settings['KeepMuteWhenTakingBack'] ?? true;
-    Config.screenReaderSupport = settings['ScreenReaderSupport'] ?? false;
-    Config.aiMovesFirst = settings['AiMovesFirst'] ?? false;
-    Config.aiIsLazy = settings['AiIsLazy'] ?? false;
-    Config.skillLevel = settings['SkillLevel'] ?? 1;
-    Config.moveTime = settings['MoveTime'] ?? 1;
-    Config.isAutoRestart = settings['IsAutoRestart'] ?? false;
-    Config.isAutoChangeFirstMove = settings['IsAutoChangeFirstMove'] ?? false;
-    Config.resignIfMostLose = settings['ResignIfMostLose'] ?? false;
-    Config.shufflingEnabled = settings['ShufflingEnabled'] ?? true;
-    Config.learnEndgame = settings['LearnEndgame'] ?? false;
-    Config.openingBook = settings['OpeningBook'] ?? false;
-    Config.algorithm = settings['Algorithm'] ?? 2;
-    Config.drawOnHumanExperience = settings['DrawOnHumanExperience'] ?? true;
-    Config.considerMobility = settings['ConsiderMobility'] ?? true;
-    Config.developerMode = settings['DeveloperMode'] ?? false;
-    Config.experimentsEnabled = settings['ExperimentsEnabled'] ?? false;
+    Config.toneEnabled = settings['ToneEnabled'] as bool? ?? true;
+    Config.keepMuteWhenTakingBack =
+        settings['KeepMuteWhenTakingBack'] as bool? ?? true;
+    Config.screenReaderSupport =
+        settings['ScreenReaderSupport'] as bool? ?? false;
+    Config.aiMovesFirst = settings['AiMovesFirst'] as bool? ?? false;
+    Config.aiIsLazy = settings['AiIsLazy'] as bool? ?? false;
+    Config.skillLevel = settings['SkillLevel'] as int? ?? 1;
+    Config.moveTime = settings['MoveTime'] as int? ?? 1;
+    Config.isAutoRestart = settings['IsAutoRestart'] as bool? ?? false;
+    Config.isAutoChangeFirstMove =
+        settings['IsAutoChangeFirstMove'] as bool? ?? false;
+    Config.resignIfMostLose = settings['ResignIfMostLose'] as bool? ?? false;
+    Config.shufflingEnabled = settings['ShufflingEnabled'] as bool? ?? true;
+    Config.learnEndgame = settings['LearnEndgame'] as bool? ?? false;
+    Config.openingBook = settings['OpeningBook'] as bool? ?? false;
+    Config.algorithm = settings['Algorithm'] as int? ?? 2;
```

Comment:
The magic number is rather ugly the way I wrote it before. I want to ask you if it is possible to store enumeration here?